packages/ns-plug/files/distfeed-setup

```diff
@@ -5,15 +5,25 @@
 # SPDX-License-Identifier: GPL-2.0-only
 #
 
-base_url=${BASE_URL:='https://updates.nethsecurity.nethserver.org'}
+base_url=${BASE_URL:=''}
```

Comment:
We could totally remove this line

Reply:
Dunno what scripts actually use the manual env set of `BASE_URL`, left it there just in case :shrug:

Reply:
we can just remove the support of env var
```suggestion
base_url="$(uci -q get ns-plug.config.repository_url)"
```